packages/google_sign_in/google_sign_in/test/google_sign_in_test.dart

```diff
@@ -9,6 +9,10 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:google_sign_in/google_sign_in.dart';
 import 'package:google_sign_in/testing.dart';
 import 'package:google_sign_in_platform_interface/google_sign_in_platform_interface.dart';
+import 'package:mockito/mockito.dart';
+
+/// Verify that [GoogleSignInAccount] can be mocked even though it's unused
+class MockGoogleSignInAccount extends Mock implements GoogleSignInAccount {}
```

Comment:
This is failing analysis. Since it's a mock, I think the best option is just to ignore `must_be_immutable` for this line.